tests/roles/development_environment/tasks/main.yaml

```diff
@@ -6,5 +6,5 @@
     EDPM_PRIVATEKEY_PATH: "{{ edpm_privatekey_path }}"
   ansible.builtin.shell:
     cmd: |
-      alias openstack="ssh -i {{ edpm_privatekey_path }} root@{{ edpm_node_ip }} OS_CLOUD=standalone openstack"
+      alias openstack="ssh -i {{ edpm_privatekey_path }} root@{{ edpm_node_ip }} OS_CLOUD={{ os_cloud_name }} openstack"
```

Comment:
add a new var to point to underclou dhere... keep that one for the compute node.